### ISSUE TITLE
EteSync v1.4.9 - Error adding member

### ISSUE THREAD

**Issue by ammarraisafti** [etesync/android#79]
# Actual behaviour
When I want to adding members, that ability must be able to be done.

# Expected behaviour
I can't add members, even though I have tried doing it repeatedly.  And l get the notifications "Error adding member".

# Step to reproduce
- Install and open the application

- Sign In using the account that you previously created. If you do not have an account, you must register first.

- Click on the account that you have created, then click on the "default" feature.  Next, click on the manage member icon at the top.

- Then click on the "add member" icon or feature, enter the email you have added, then select OKAY.

- Then you will find problems like I have experienced.

# Information
- Device: Nubia n2 lite
- Android version: 8.1.0
- Application version: 1.4.9

# Recording of the bug
https://youtu.be/C-y5b_LbY7s

**Comment by tasn** [comment]
This is not an issue with EteSync but rather with your setup. As you can see by the error message, it's failing to connect to 127.0.0.1:8118 which is most likely a proxy you have set up (or another app that filters network traffic) that is malfunctioning. Please check that, but it's definitely not a problem with the app, which is working correctly. This is the same reason why you couldn't create in #78 (but unrelated to the total application crash you mentioned).

So in summary, please check, maybe you have a proxy set up in the EteSync app or system/wifi wide, or maybe you have an app that's trying to redirect traffic.
